cli/src/pcluster/config/cluster_config.py

```diff
@@ -2214,6 +2233,14 @@ def _register_validators(self, context: ValidatorContext = None):
                 queue_name=self.name,
                 subnet_ids=self.networking.subnet_ids,
             )
+        if bool(self.custom_slurm_settings):  # if not empty register validator
```

Comment:
Nitpick: `bool()` should not be necessary here.

Reply:
It is.
I had to add it ..without it doesn't work with empty dictionaries.

Reply:
That's weird. I'm not getting this behavior locally:
```
In [1]: A = {}

In [2]: if A:
   ...:     print("yes")
   ...: 

In [3]: B = {"bla": "blu"}

In [4]: if B:
   ...:     print("yes")
   ...: 
yes

```